attributes/ldap_config.rb

```diff
@@ -1,4 +1,9 @@
 default['grafana']['ldap_verbose_logging'] = false
+default['grafana']['ldap_multi_tree_auth'] = false
+# When the ldap_multi_tree_auth attribute is set to true, it is possible to configure
```

Comment:
There is no way to make default['grafana']['ldap']['[servers]'] hold a list of hashes ?
Not sure it would be easy to override ...